acj/static/modules/common/avatar-directive.js

```diff
@@ -6,12 +6,13 @@
 			scope: {
 				userId: '=',
 				avatar: '=',
+                forceDefault: '=?',
 				displayName: '=?',
 				fullName: '=?',
 				me: '=?'
 			},
 			template: '<a ng-href="#/user/{{ userId }}">' +
-						'<img src="//www.gravatar.com/avatar/{{ avatar }}?s=32&d=retro" /> ' +
+						'<img src="//www.gravatar.com/avatar/{{ avatar }}?s=32&d=retro{{ forceDefault ? \'&f=y\' : \'\' }}" /> ' +
```

Comment:
The chance of collision is very small.

If we do use a default image for all students, we could use an static image instead of gravatar since gravatar wouldn't be doing special for students anymore.